packages/fxa-profile-server/lib/routes/avatar/post-from-auth-server.js

```diff
@@ -0,0 +1,65 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const assert = require('assert');
+
+const Joi = require('@hapi/joi');
+
+const config = require('../../config');
+const db = require('../../db');
+const hex = require('buf').to.hex;
+const img = require('../../img');
+const notifyProfileUpdated = require('../../updates-queue');
+const workers = require('../../img-workers');
+const avatarShared = require('./_shared');
+
+const FXA_PROVIDER = 'fxa';
+const FXA_URL_TEMPLATE = config.get('img.url');
+assert(
+  FXA_URL_TEMPLATE.indexOf('{id}') !== -1,
+  'img.url must contain the string "{id}"'
+);
+const DEFAULT_AVATAR_ID = config.get('img.defaultAvatarId');
+assert(DEFAULT_AVATAR_ID.length === 32, 'img.default');
```

Comment:
Q: do you want `assert.strictEqual(DEFAULT_AVATAR_ID.length, 32, 'img.defaultAvatarId')` here?